vipul/Gallery_System/style.css

```diff
@@ -0,0 +1,274 @@
+body{
+	margin:10px;
+	position: relative;
+}
+
+.container{
+	border-bottom:1px solid #909595;
+}
+
+.active{
+	color:#06D6A0;
+}
+
+.header{
+	position: absolute;
+	top: 0;
+	margin-left:20px;

```

Comment:
why did you use margin-left property for this class?

Reply:
I used heading for the page with h1 tag. by default it will displayed to left side of screen so i want  some space from left side. I used margin-left property.